src/components/BoxApproval/BoxApproval.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { Stack, Button } from '@chakra-ui/react';
+import Box from './Box';
+import './BoxApproval.css';
+
+function BoxApproval() {
+  const [currentTab, setCurrentTab] = useState('requiresApproval');
+  // eslint-disable-next-line no-unused-vars
+  const [requiresApprovalBoxes, setRequiresApprovalBoxes] = useState([Box(), Box(), Box()]);
+  // eslint-disable-next-line no-unused-vars
+  const [approvedDeniedBoxes, setApprovedDeniedBoxes] = useState([Box(), Box()]);
+
+  return (
+    <div className="boxApproval">
+      <h1 className="boxApprovalHeading">Box Approval</h1>
+      <div className="boxApprovalMenu">
+        <Stack spacing={0} direction="row" align="center" className="buttonStack">
+          <Button
+            onClick={() => setCurrentTab('requiresApproval')}
+            size="xs"
+            border="none"
+            width="100%"
```

Comment:
Currently, these two buttons show up as longer than the box approval component on my screen. I believe changing the width for both of these to 50% should fix this issue. Please try to resize your screen to make sure this (and other styling) looks good after this change.